packages/@uppy/provider-views/src/View.js

```diff
@@ -68,7 +59,7 @@ export default class View {
   // todo document what is a "tagFile" or get rid of this concept
   getTagFile (file) {
     const tagFile = {
-      id: providerFileToId(file), // todo what's this used for? can be removed?
+      id: file.id,
```

Comment:
Shall we make uppy.addFile(s) accept id then, too? Gotta make sure they don't change on the fly with providers.